src/LIBMONGOC_VERSION_CURRENT

```diff
@@ -1 +1 @@
-1.22.1
+1.11.1-20220829+git623d659f00
```

Comment:
This looks wrong. I don't recall how I used to fix it - the problem either appeared when checking out a specific commit hash or when checking out the master branch. Either way, the version should definitely not be 1.11 ;)

Reply:
This has long been tracked in [CDRIVER-3315](https://jira.mongodb.org/browse/CDRIVER-3315). When you've updated this file in the past, did you manually change `1.11.1` to a particular version?

I've generally ignored this and haven't pushed for it to be fixed upstream since our releases always use actual libmongoc tags.

Reply:
I think I cheated and updated it manually. Either way, fine to have it as is 👍